make/common/Modules.gmk

```diff
@@ -231,6 +231,16 @@ ifeq ($(call isTargetOs, windows macosx linux), false)
   MODULES_FILTER += jdk.incubator.jpackage
 endif
 
+################################################################################
+# Filter out specific modules for mobile platforms
+
+ifeq ($(call isTargetOs, ios android), true)
+  MODULES_FILTER += jdk.hotspot.agent
+  MODULES_FILTER += jdk.management
+  MODULES_FILTER += jdk.management.agent
+  MODULES_FILTER += jdk.sctp
```

Comment:
While technically correct, I agree with Bob that there seem to be not much point in disabling these, if the reason is just "don't think we need them". Is there an issue with compiling these modules on mobile?

Reply:
The compile issues were minimal, and they are solved now.